### PR TITLE
update to latest locations for download

### PR DESCRIPTION
**Pull request by Takadonet**
## Description of changes
Update documentation to reflect new location of public datasets and virtualbox

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [ ] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
* [ ] Tests added (or description of how to test) for any new features.
* [x] User documentation updated for UI or technical changes.

### CODE REVIEW

File: doc/developer/index.md [hunk 1]
```diff
@@ -26,7 +26,6 @@ Tool Development
 
 User Interface development
 --------------------------
-* [User Interface development](interface)
```
Comment (tom114) [inline doc/developer/index.md:29]: Remove this change please.  Looks like it was an accidental deletion.

**Review by tom114 — CHANGES_REQUESTED** [review]

**Review by apetkau — APPROVED** [review]
Looks good to me (other than Tom's change).

File: doc/downloads/index.md [hunk 2]
```diff
@@ -64,7 +64,7 @@ Sample Data
 
 Sample data for testing IRIDA can be found at:
 
-<https://irida.corefacility.ca/downloads/data/irida-sample-data.zip>
+<https://sairidapublic.blob.core.windows.net/downloads/data/irida-sample-data.zip>
 
 The sample data package consists of *simulated* Illumina MiSeq read data for three publicly available Listeria Monocytogenes genomes. You can find more information about the simulated data at:
```
Comment (tom114) [inline doc/downloads/index.md:69]: Below this line there's also a `README.txt` link to be updated.

File: doc/developer/index.md (hunk 1 above)
Comment (Takadonet) [reply doc/developer/index.md]: Added back.

File: doc/downloads/index.md (hunk 2 above)
Comment (Takadonet) [reply doc/downloads/index.md:69]: Fixed

**Review by tom114 — APPROVED** [review]

**Pull request merged**